File: src/test/java/no/nav/syfo/LocalApplication.java
package no.nav.syfo;

import no.nav.security.spring.oidc.api.EnableOIDCTokenValidation;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Profile;

@SpringBootApplication
@EnableOIDCTokenValidation(ignore="org.springframework")
@Profile("local")
public class LocalApplication {
    public static void main(String[] args) {
        SpringApplication.run(LocalApplication.class, args);
    }
}

File: src/test/kotlin/no/nav/syfo/LocalApplication.kt
package no.nav.syfo

import no.nav.security.token.support.spring.api.EnableJwtTokenValidation
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Profile

@SpringBootApplication
@EnableJwtTokenValidation
@Profile("local")
class LocalApplication

fun main(args: Array<String>) {
    runApplication<LocalApplication>(*args)
}
